tests/test_mcp_workspace_git.py

```
@@ -0,0 +1,777 @@
+    """Local bare git repo with an initial commit on 'main'.
+
+    Structure:
+        README.md       — "# Test Repo\\n\\nInitial content."
+        src/main.py     — hello() + add() stubs
+    """
+    bare = tmp_path / "origin.git"
+    work = tmp_path / "_seed"
+    work.mkdir()
+
+    # Bare repo, force 'main' as default branch
+    subprocess.run(["git", "init", "--bare", str(bare)],
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)